### DISCUSSION TITLE
Check out rendercv-pipeline

### DISCUSSION THREAD

**Discussion by sinaatalay** [rendercv/rendercv#79] [Announcements]
> Thanks to @c2tz, for the inspiration with his [work](https://github.com/c2tz/cv).

Are you ready to revolutionize the way you manage and craft your CVs and resumes?

Picture this:

1.  You simply update your `John_Doe_CV.yaml` input file and push the changes.
2.  Then, a pipeline generates a new PDF and Markdown from it and uploads it to the repository.
3.  When you are done with your work, you can create a new release on GitHub, tagging your CV with something like `v2024-04`. The pipeline will automatically add the PDF and the YAML as assets to the release.
4.  You have successfully created your CV pipeline, and all the history is in safe hands and reproducible.

**How to start?**

1.  [Click here](https://github.com/new?template_name=rendercv-pipeline&template_owner=sinaatalay) to create your CV repository based on the [`rendercv-pipeline`](https://github.com/sinaatalay/rendercv-pipeline) template repository.
2.  Edit the `John_Doe_CV.yaml` file and push the changes. Then, see the magic happen.

**Comment by c2tz** [comment]
Thanks for spying on me ! 🫡
